pkg/disk/utils.go

```diff
@@ -629,7 +629,9 @@ func getDiskVolumeOptions(req *csi.CreateVolumeRequest) (*diskVolumeArgs, error)
 			diskVolArgs.FsType = EXT4_FSTYPE
 		}
 	}
-	if diskVolArgs.FsType != EXT4_FSTYPE && diskVolArgs.FsType != EXT3_FSTYPE && diskVolArgs.FsType != XFS_FSTYPE {
+
+	supportedFilesystemTypes := sets.New(EXT4_FSTYPE, EXT3_FSTYPE, XFS_FSTYPE, NTFS_FSTYPE)
```

Comment:
Should `supportedFilesystemTypes` be a global variable? Since it is readonly. We don't need to build the hash table every time.